envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/NodeMetadataTest.kt

```diff
@@ -63,7 +64,7 @@ class NodeMetadataTest {
         val proto = outgoingDependencyProto()
 
         // expects
-        val exception = assertThrows<NodeMetadataValidationException> { proto.toDependency() }
+        val exception = assertThrows<NodeMetadataValidationException> { proto.toDependency(SnapshotProperties()) }
```

Comment:
This is repeated a lot in tests. Can `toDependency` extension function have a default parameter of `SnapshotProperties()` so you wouldn't need to modify the tests?